internal/bundle/helpers.go

```diff
@@ -13,6 +14,13 @@ import (
 	"github.com/databricks/cli/libs/cmdio"
 	"github.com/databricks/cli/libs/flags"
 	"github.com/databricks/cli/libs/template"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/compute"
+	"github.com/databricks/databricks-sdk-go/service/files"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+	"github.com/stretchr/testify/require"
```

Comment:
this will make `testify` part of the resulting binary. you should put this file into other package, like `bundle_test`

Reply:
This package is used for testing only. Agree it would be good to rename the package/directory to indicate that.